src/99-appendices/04-entity-table.md

```diff
@@ -39,7 +39,7 @@ while entity definitions are in [Appendix IX](09-entities.md).
 | meg<br>(meg)               | REQUIRED                            | OPTIONAL                            | REQUIRED                              | OPTIONAL                            | OPTIONAL                            | OPTIONAL                              |                                         | OPTIONAL                                |
 | meg<br>(headshape)         | REQUIRED                            | OPTIONAL                            |                                       | OPTIONAL                            |                                     |                                       | OPTIONAL                                |                                         |
 | meg<br>(markers)           | REQUIRED                            | OPTIONAL                            | OPTIONAL                              | OPTIONAL                            |                                     |                                       | OPTIONAL                                |                                         |
-| channels<br>(meg eeg ieeg) | REQUIRED                            | OPTIONAL                            | REQUIRED                              |                                     | OPTIONAL                            |                                       |                                         |                                         |
+| channels<br>(meg eeg ieeg) | REQUIRED                            | OPTIONAL                            | REQUIRED                              | OPTIONAL                            | OPTIONAL                            |                                       |                                         |                                         |
```

Comment:
The entity table shouldn't be edited directly. It should be generated from the `bids_schema.py` script.

Reply:
OK, sorry about that. I restored the original `99-entity-table.md`.